### PR TITLE
M4W1 Assessment

### PR DESCRIPTION
**Pull request by zoefarrell**

### CODE REVIEW

File: module4/assessments/Week1-3021.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+</aside>
+
+### Submission and Feedback
+When completed, click the `Invite` button in the upper right corner and `Generate a join link`.  Include that link in your form submission.
+
+![](/assets/images/module1/Week1/GenerateJoinLink.png)
+
+## Part 1: Questions (10 Points Total)
+
+Answer the questions in the `questions.md` file in the Replit.
```
Comment (rtillies) [inline module4/assessments/Week1-3021.md:22]: I like the questions:

1. STAR method question is perfect.
2. Debug question is perfect.
3. Have we asked students to perform brand new research during an assessment before?
4. This is all or nothing, I presume. Do they need to get all 5 correct to earn the point?

File: module4/assessments/Week1-3021.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Part 2: Build a Quiz Question (10 Points Total)
+
+Your goal is to build off of the starter code to replicate the behavior in the following video.
+<video controls width="800">
+    <source src="https://github.com/turingschool/launch-curriculum/assets/11747682/80e470c0-611d-4be9-aaed-f8b7f8dea095" type="video/mp4">
+    Your browser does not support the video tag.
+</video>
```
Comment (rtillies) [inline module4/assessments/Week1-3021.md:30]: The video doesn't show up here but I grabbed the link from the `<video>` tag. I think I paid more attention to your bookmarks to make sure you didn't display anything untoward LOL.

I also notice you included the `.mp4` object but don't use it. Are they the same video?

**Review by rtillies — APPROVED** [review]
I like!

File: module4/assessments/Week1-3021.md (hunk 2 above)
Comment (zoefarrell) [reply module4/assessments/Week1-3021.md:30]: Good call on the .mp4 video, that was from my first attempt at including a video file before I decided to host it on Github, which seems to work well. Will remove the old file.

Thanks for the second set of eyes on the bookmarks, but they should all be bland launch stuff. lol.

File: module4/assessments/Week1-3021.md (hunk 1 above)
Comment (zoefarrell) [reply module4/assessments/Week1-3021.md:22]: For 3. We have had them google small things in the assessments before, and it's feeling ok to me to lean more into that as we move later in the course and release responsibility.

For 4.  I'm not following what you mean by "get all 5 correct" mean. I have 4 written questions in the assessment and I put the max point values for each in the Repl. I'm imagining we can also do partial points for the multiple-point questions. I'm going to merge this PR in so it's ready for next week, but if you have other concerns or suggestions for changing the grading structure let me know! We can always come back and make an edit.

Comment (rtillies) [reply module4/assessments/Week1-3021.md:22]: Question 4 is "What are 5 different types of events in Javascript?". It is worth 1 point. Do they need to name all 5 types correctly to get the point?

**Pull request merged**